src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const transformToNumber = (str) => {
+  return parseFloat(str.replace(/[$,]/g, ''));
+};
+const getEmployees = (list) => {
+  return list.map((employee) => {
+    return {
+      name: employee.textContent.trim(),
+      salary: transformToNumber(employee.getAttribute('data-salary')),
+    };
+  });
+};
+const SortList = (list) => {
+  return list.sort((a, b) => b.salary - a.salary);
+};
+
+const employeesArray = Array.from(document.querySelectorAll('ul li'));
+const employeesData = employeesArray.map((employee) => {
+  getEmployees(employee);
+});
+
+SortList(employeesData);
```

Comment:
The `SortList` function is called, but its result is not used or stored. You should assign the sorted list to a variable if you intend to use it later.